src/test/resources/authz.json

```diff
@@ -31,6 +31,17 @@
         "support"
       ]
     },
+    {
+      "route": "getTenantByExternalId",
```

Comment:
this requires also the test in the spec

Reply:
I prefer to add this test after merged with PIN-3494 because I have created an ad-hoc spec for this purpose.

Reply:
can you move the PR to draft then? 🙏

Reply:
Resolved!